tests/test_sqla_yaml_fixtures.py

```diff
@@ -499,3 +526,29 @@ def test_custom_loader(session):
     assert users[0].email == 'deedee@ramones.org'
     assert users[1].username == 'joey'
     assert users[1].email == 'joey@ramones.org'
+
+
+def test_many_2_many(session):
+    fixture = """
+- User:
+  - __key__: joey
+    username: joey
+    email: joey@example.com
+  - __key__: bobby
+    username: bobby
+    email: bobby@example.com
+  - __key__: johnny
+    username: johnny
+    email: johnny@example.com
+    friends:
+      - joey
+      - bobby
+"""
+    sqla_yaml_fixtures.load(BaseModel, session, fixture)
+    users = session.query(User).all()
+    for u in users:
```

Comment:
Please keep the tests "stupid" simple. You should not use FOR-loops or IF conditions on tests.